portality/templates/publisher/metadata.html

```diff
@@ -28,13 +28,20 @@ <h4 class="red">There is a problem with the submitted form.</h4>
                         <button type="submit" id="remove_{{subfield.short_name}}" name="remove_{{subfield.short_name}}" class="btn btn-danger remove_button">
                           <i class="glyphicon glyphicon-remove"></i>
                         </button>
+                        <div class="error">
```

Comment:
@richard-jones  This div here is necessary to inherit red color for subfield.errors, tbh I don't entirely understand why it works like this. There would probably be some more elegant way to do this if we would rebuild CSS but this way it's much easier and less time-consuming. Maybe we could have a look at it later?

Reply:
It doesn't seem to be causing the red colour to be inherited even so - on my screen the error came up in black text.  We can look at this together, yes.

Reply:
It was. The red color disappeared after changing the orientation of the fields.

![image](https://user-images.githubusercontent.com/8298769/70317193-30c7c600-1815-11ea-974f-8f73a2eb8678.png)